make/launcher/LauncherCommon.gmk

```diff
@@ -107,6 +107,16 @@ define SetupBuildLauncherBody
       $1_LDFLAGS += -Wl,-Ttext-segment=0x60000
     endif
   endif
+  
+  # Set text/data load address for x64_aix to 0x100,0000,0000 (1TB) for launchers,
```

Comment:
Please fix the comment, similarly to the jdk8 change. For jdk13 and jdknext as well if necessary.

Reply:
Done for JDK11, 13 and Next